GitUI/UserControls/FilterToolBar.cs

```diff
@@ -27,7 +27,9 @@ public FilterToolBar()
             // Select an option until we get a filter bound.
             SelectShowBranchesFilterOption(selectedIndex: 0);
 
-            tstxtRevisionFilter.Items.AddRange(AppSettings.RevisionFilterDropdowns);
+            tstxtRevisionFilter.Items.AddRange(AppSettings.RevisionFilterDropdowns
+                .Union([@"--invert-grep --grep=""EXCLUDE_COMMIT_MESSAGE_REGEX_PATTERN""", @"--perl-regexp --author=""^(?!.*EXCLUDE_AUTHOR_REGEX_PATTERN)"""])
```

Comment:
>It is not doable easily because it conflicts with already in place branch GE filtering.

Exclude remotes suggestion
```suggestion
                .Union([@"--invert-grep --grep=""EXCLUDE_COMMIT_MESSAGE_REGEX_PATTERN""", @"--perl-regexp --author=""^(?!.*EXCLUDE_AUTHOR_REGEX_PATTERN)""", @"--exclude=refs/remotes/EXCLUDE_REMOTE_REGEX_PATTERN"])
```

Reply:
Added.